common/src/main/java/org/jboss/hal/testsuite/testlistener/ServerStateListenerOperations.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2016, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.hal.testsuite.testlistener;
+
+import static java.util.function.Function.identity;
+import static java.util.stream.Collectors.toMap;
+import static org.jboss.as.controller.client.helpers.ClientConstants.HOST;
+import static org.jboss.as.controller.client.helpers.ClientConstants.SERVER;
+import static org.jboss.as.controller.client.helpers.ClientConstants.SERVER_CONFIG;
+import static org.jboss.as.controller.client.helpers.ClientConstants.STATUS;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import org.jboss.hal.testsuite.creaper.ManagementClientProvider;
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * Created by pjelinek on Apr 19, 2016
+ */
+public class ServerStateListenerOperations {
+
+    /**
+     * Has to be initialized in every public method which requires it with new {@link OnlineManagementClient}
+     * which is closed before the method finishes.
+     */
+    private Operations ops;
```

Comment:
If I am looking correctly, you are using the `ops` to actually something which is provided between methods, correct?